src/components/dev-height-adjustable-placeholder/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react'
+
+function DevHeightAdjustablePlaceholder({
```

Comment:
Found this useful in testing, but will delete along with `/test-sidecar-layout` page before merging 👍